OPM: child registration fix

lwyszomi opened the pull request:
@czue @esoergel 

last comments in case: http://manage.dimagi.com/default.asp?187634

Commit:
OPM: child registration fix

esoergel commented:
Test failure looks real, likely just needs to be updated
```
======================================================================
FAIL: test_service_unavailable_at_all (custom.opm.tests.test_child_condition_four.TestChildBirthRegistration)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/custom/opm/tests/test_child_condition_four.py", line 96, in test_service_unavailable_at_all
    self.assertEqual(True, self.get_condition(row))
AssertionError: True != False
```

Commit:
fixed tests

czue commented:
still failing (on a different test). maybe run them locally so you don't have to wait for travis?
```
======================================================================
FAIL: test_service_unavailable_different_condition (custom.opm.tests.test_child_condition_four.TestChildBirthRegistration)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/custom/opm/tests/test_child_condition_four.py", line 131, in test_service_unavailable_different_condition
    self.assertEqual(True, self.get_condition(row))
AssertionError: True != False
----------------------------------------------------------------------
```

lwyszomi commented:
@czue 
Sure I run them locally. I thought that the travis shows me the all failing tests.

czue commented:
ah, no it uses --failfast

lwyszomi commented:
@czue now I know :)

Commit:
fixed test v2

lwyszomi commented:
@czue @esoergel 
I fixed tests, I removed the ```ServiceAvailabilityTestMixIn``` from ```TestChildBirthRegistration``` because currently in method ```child_birth_registered``` we don't check service.

esoergel commented on custom/opm/tests/test_child_condition_four.py:
```diff
@@ -151,7 +151,7 @@ def setUp(self):
 
 
-class TestChildBirthRegistration(OPMCaseReportTestBase, ConditionFourTestMixin, ServiceAvailabilityTestMixIn):
+class TestChildBirthRegistration(OPMCaseReportTestBase, ConditionFourTestMixin):
```
:star2:
